//python -m grpc_tools.protoc -I ./common_proto_files --python_out=. --grpc_python_out=. ./common_proto_files/greeting.proto
syntax = "proto3";

service MyStudents {
   rpc student_details (Student_ID) returns (Student_Data);
   rpc student_classes (Student_ID) returns (Class_data);
}

message Student_ID{
   string student_Id_number = 1;
}

message Student_Data{
   bool result = 1;
   string name = 2;
   string surname = 3;
   string Date_of_birth = 4;
}
message Class_data{
   string class_name =1;
   string teacher_name =2;
}

